### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by svitalis123**
# Bookstore: styling
## I accomplished

- [ ] Styling the Bookstore app

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -20,21 +21,26 @@ const Form = () => {
...
+          onChange={(e) => setCategory(e.target.value)}
+          placeholder="category"
+        >
+          <option value="Leadership">Leadership</option>
+          <option value="Science">Science</option>
+          <option value="Fiction">Fiction</option>
+        </select>
+        <button type="submit" className={styles.container_form_form_button}>Add Book</button>
+      </form>
+    </div>
```
Comment (thecodechaser) [inline src/components/Form.js:43]: - [ ] Please make sure that all form elements including the add button are on a single row as on the Zeplin design.

### Your design
image

### Zeplin design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @svitalis123👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js (hunk 1 above)
Comment (svitalis123) [reply src/components/Form.js:43]: thank you

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @svitalis123👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**